grouper/fe/handlers/service_account_create.py

```diff
@@ -33,6 +33,13 @@ def post(self):
                 alerts=self.get_form_alerts(form.errors)
             )
 
+        if form.data["name"].split("@")[-1] != settings.service_account_email_domain:
```

Comment:
Can we just have the user select the local part of the email address and append the hostname to it?